README.md

```diff
@@ -1,5 +1,21 @@
-# Python boilerplate for GitHub tasks
+# Serializer Implementation
 
 - **Please note:** read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md)
-before starting.
-- Implement the task described [here](app/main.py)
+before start.
+
+In this task, you should implement a serializer `CarSerializer` for the `Car` model in `serializers.py`.
```

Comment:
In this task, you should implement the `CarSerializer` serializer for the `Car` model in the `serializers.py` file.